lib/screenplay_web/controllers/alerts_api_controller.ex

```diff
@@ -2,9 +2,17 @@ defmodule ScreenplayWeb.AlertsApiController do
   use ScreenplayWeb, :controller
 
   alias Screenplay.Alerts.Alert
+  alias Screenplay.Alerts.ScreensByAlert
 
   def index(conn, _params) do
     {:ok, alerts} = Alert.fetch()
-    json(conn, Enum.map(alerts, &Alert.to_full_map/1))
+
+    {:ok, screens_by_alerts} =
+      alerts |> Enum.map(& &1.id) |> ScreensByAlert.get_screens_by_alert()
```

Comment:
A big advantage to having all caching logic live in the screens server is it gives us a super easy way to set everything up locally. If screenplay always communicated with memcache, it would have to look at data from dev-green or dev because local would not write to memcache. A very specific advantage to us devs but still helps a lot. 

And I think you are right about that downside being ok. Screenplay's goal is to represent the screens in a way that assumes all hardware is working as it should. Not the case for software. So if the screens app is down, the simulations won't work/counts won't show and PIOs will be able to see first hand that there is a problem on live screens.